### DISCUSSION TITLE
nesting module

### DISCUSSION THREAD

**Discussion by azar-maghsoodi** [area17/twill#2239] [Q&A]
could you please tel me how we can write indexdata() function in the controller when we have self nested module?

**Comment by ifox** [comment]
Hi @azar-maghsoodi unfortunately self-nested modules listings are limited in functionality, because of the nesting, actually. Sorting wouldn't make sense with multiple levels of nesting (beyond the first level). One could argue that columns could be supported, but the indentation of the nested items makes it look weird. If you really need it, it is certainly possible with an override of Vue components, let us know if you are going in that direction and need support.

**Reply by azar-maghsoodi** [reply]
Thank you for the response, I want to have sort able and optional with self nesting, could you tell me please how to start to do that with Vue component,I mean give me some hint.I want to do sort in parnet level and to have optional column.
Thank you in advance.

**Comment by ifox** [comment]
To clarify though, what are you trying to do with indexData exactly?
